Makefile

```diff
@@ -68,8 +68,11 @@ docs: install.tools ## build the docs on the host
 gopath:
 	test $(shell pwd) = $(shell cd ../../../../src/github.com/containers/buildah ; pwd)
 
+codespell:
+	codespell -S build,buildah,imgtype,AUTHORS,bin,vendor,.git,go.sum,CHANGELOG.md,changelog.txt,seccomp.json,.cirrus.yml,"*.xz,*.gz,*.tar,*.tgz,*ico,*.png,*.1,*.5,*.orig,*.rej" -L uint,iff,od
+
 .PHONY: validate
-validate: install.tools
+validate: install.tools codespell
```

Comment:
same concern here as in Podman for having this as a gate rather than a warning.

Reply:
I have no problem if we move it down from a requirement, if it proves to be a pain in the ass, but in my opinion for now, it is not a hardship.